projects/core/src/user/facade/user.service.ts

```diff
@@ -22,7 +23,15 @@ import {
 
 @Injectable()
 export class UserService {
-  constructor(protected store: Store<StateWithUser | StateWithProcess<void>>) {}
+  /**
+   * @deprecated since version 1.x
+   *  Use constructor(el: ElementRef, renderer: Renderer2) instead
+   */
```

Comment:
We don't know yet which version the ASM feature will be released.  We have GH-4075 as a reminder for that.